### ISSUE TITLE
Bugs View ist leer

### ISSUE THREAD

**Issue by GoogleCodeExporter** [hansriezebos/projectusus#13]
```
Trotz zweier eingetragener Bugs ist der Bugs View leer, und der
Average-Anzeiger zeigt auch keine Bugs an.
```

Original issue reported on code.google.com by `nicol...@gmx.de` on 20 Nov 2009 at 1:07

**Comment by GoogleCodeExporter** [comment]
```
At the moment there are no listeners for new bugs.
You can fill Bugs View with Refresh.

I will implement a listener for new bugs.

```

Original comment by `s.w.schu...@googlemail.com` on 24 Nov 2009 at 8:08
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Fixed. Fix will be available at version 0.4.0
```

Original comment by `s.w.schu...@googlemail.com` on 4 Jan 2010 at 10:12
- Changed state: **Verified**
